src/loaders/stm_load.c

```diff
@@ -131,12 +159,48 @@ static int stm_load(struct module_data *m, HIO_HANDLE * f, const int start)
 	sfh.type = hio_read8(f);	/* 1=song, 2=module */
 	sfh.vermaj = hio_read8(f);	/* Major version number */
 	sfh.vermin = hio_read8(f);	/* Minor version number */
-	sfh.tempo = hio_read8(f);	/* Playback tempo */
-	sfh.patterns = hio_read8(f);	/* Number of patterns */
-	sfh.gvol = hio_read8(f);	/* Global volume */
-	hio_read(&sfh.rsvd2, 13, 1, f);	/* Reserved */
+	version = (100 * sfh.vermaj) + sfh.vermin;
+
+	if (version == 100 || version > 221) {
```

Comment:
Should probably be more tight and reject unknown versions not found in the wild?

Reply:
Here you can download STM song of version 1.10: https://files.fm/u/xe3g4ncq
This one was ripped from FCSLIDE1 demo. This is the earliest STM that I have seen.
So I offer to check for fixed list of version numbers: 100 (I will implement decoding of patterns data), 110, 200, 210, 220 and 221.

Reply:
Sounds reasonable to me.